photon-api/src/main/scala/com/linkedin/photon/ml/data/LocalDataSet.scala

```diff
@@ -15,6 +15,7 @@
 package com.linkedin.photon.ml.data
 
 import scala.collection.mutable
+import scala.collection.Set
```

Comment:
Drop this import - it can cause problems in other places. Sometimes the IDE gets confused an imports things like `scala.collection.Set` or `scala.collection.Map`, but these are actually different interfaces than the default `Set` and `Map` calls which can screw things up.